### ISSUE TITLE
Anti-Recoverin antibody

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10549]
**Preferred term label:**

**Synonyms**

Anti-retinal CAR antigen antibody

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against recoverin. Recoverins are calcium-binding proteins primarily expressed in retinal rod photoreceptors and primarily involved in phototransduction.

PMID: 38054667 Anti-recoverin antibodies are typically found in cancer-associated retinopathy (CAR) or autoimmune diseases. We present a case of anti-recoverin positive cerebellar syndrome without any signs of malignancy or retinopathy. The patient was treated with steroids and intravenous immunoglobulins, improving cognitive and motor symptoms.

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:71505 Cancer-associated retinopathy

Your nano-attribution (ORCID)
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000882
- label: Anti-recoverin antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:38054667
- def: The presence of autoantibodies in the serum that react against recoverin. Recoverins are calcium-binding proteins primarily expressed in retinal rod photoreceptors and primarily involved in phototransduction.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10549
- synonyms: Anti-retinal CAR antigen antibody(NONE)
